org.kde.knights.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "org.kde.knights",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "knights",
+    "rename-icon": "knights",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
```

Comment:
don't we need network so you can play in freechess.org and pulseaudio so that sound works?

Reply:
Added